### PR TITLE
Add a encrypted video stream for integration test of video_player's iOS 16 bug

### PR DESCRIPTION
**Pull request by hellohuanlin**
This is for integration test of the video_player's iOS 16 bug. Will be used in https://github.com/flutter/plugins/pull/6442. 

The key is generated using: 
```
openssl rand 16 > encrypted_bee.key
```

The `key_info` is (used locally for the ffmpeg cmd only): 
```
https://flutter.github.io/assets-for-api-docs/assets/videos/hls/encrypted_bee.key
encrypted_bee.key
```

And the m3u8 is created using: 
```
ffmpeg -i bee.mp4 -g 60 -hls_time 2 -hls_list_size 0 -hls_segment_size 500000 -hls_key_info_file key_info encrypted_bee.m3u8
```

*List which issues are fixed by this PR. You must list at least one issue.*

https://github.com/flutter/flutter/issues/111457

Also fixes https://github.com/flutter/flutter/issues/112114

*If you had to change anything in the [flutter/tests] repo, include a link to the migration guide as per the [breaking change policy].*

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read the [Flutter Style Guide] _recently_, and have followed its advice.
- [x] I signed the [CLA].
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

File: assets/videos/hls/encrypted_bee.key [hunk 1]
```diff
@@ -0,0 +1,2 @@
+���k
+hw't6���
```
Comment (hellohuanlin) [inline assets/videos/hls/encrypted_bee.key:2]: This is random key, generated with
```
openssl rand 16 > encrypted_bee.key
```

**Comment by jmagman** [comment]
See also https://github.com/flutter/assets-for-api-docs/pull/178 for similar PR.

Analyzer warnings are unrelated, filed https://github.com/flutter/flutter/issues/112114

**Review by jmagman — APPROVED** [review]
LGTM once analyzer is happy on deprecated usage

**Comment by hellohuanlin** [comment]
theres no autosubmit label. i will just manually land it

**Pull request merged**